tests/Tests/Unit/TestTrait.php

```diff
@@ -39,6 +39,8 @@
 use Google\Protobuf\Any;
 use Google\Rpc\Code;
 use Google\Rpc\Status;
+use stdClass;
```

Comment:
I can't find where this is used, why are you adding it?

Reply:
It was an oversight, I added it to test something and then forgot to remove it. This will be done in my next commit 💪